src/java/boa/runtime/BoaRunner.java

```diff
@@ -82,7 +83,9 @@ public Job job(final Path[] ins, final Path out, final boolean robust) throws IO
 		// map output compression
 		configuration.setBoolean("mapred.compress.map.output", true);
 		configuration.set("mapred.map.output.compression.type", "BLOCK");
-		configuration.setClass("mapred.map.output.compression.codec", SnappyCodec.class, CompressionCodec.class);
```

Comment:
Don't reset/unset - just *set* it again.  Set it to "org.apache.hadoop.io.compress.DefaultCodec".  Or some specific codec that ships with Hadoop (Snappy doesnt, hence the problem with this for local mode).